src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/PositionExtensions.cs

```diff
@@ -12,7 +13,7 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Extensions
 {
     internal static class PositionExtensions
     {
-        public static int GetAbsoluteIndex(this Position position, SourceText sourceText)
+        public static bool TryGetAbsoluteIndex(this Position position, SourceText sourceText, out int absoluteIndex, ILogger? logger = null)
```

Comment:
Out parameters should be last , so I'd much rather prefer a separate overload for this, or just get rid of the optional parameter. In fact I would rather we get rid of all optional parameters for internal APIs

Reply:
Good call. I forget why I thought I needed to make it optional, so I just made it required.